pypeit/specobj.py

```diff
@@ -517,6 +517,8 @@ def apply_flux_calib(self, wave_zp, zeropoint, exptime, tellmodel=None, extinct_
                 Used for extinction correction
             extrap_sens (bool, optional):
                 Extrapolate the sensitivity function (instead of crashing out)
+            sens_fwhm (`numpy.ndarray`_, optional):
```

Comment:
Let's remove this sens_fwhm functionality and only add it if it is being used.

Reply:
removed.